extensions/amp-form/0.1/amp-form.js

```diff
@@ -291,7 +319,16 @@ export class AmpForm {
       xhrUrl = addParamsToUrl(url, values);
     } else {
       xhrUrl = url;
-      body = createFormDataWrapper(this.win_, this.form_);
+      if (this.encType_ === 'application/x-www-form-urlencoded') {
+        body = serializeQueryString(this.getFormAsObject_());
+        headers = dict({
+          'Accept': 'application/json',
+          'Content-Type': 'application/x-www-form-urlencoded',
+        });
+      } else {
+        // default case: encType_ is 'multipart/form-data'
```

Comment:
Nit: assert to enforce this/catch issues

```suggestion
        // default case: encType_ is 'multipart/form-data'
        devAssert(this.encType_ === 'multipart/form-data');
```

Reply:
Thanks Alan, I've added this change!